exercises/difference-of-squares/src/example.h

```diff
@@ -0,0 +1,8 @@
+#ifndef _DIFFERENCE_OF_SQUARES_H
+#define _DIFFERENCE_OF_SQUARES_H
+
+unsigned int sum_of_squares(unsigned int number);
+unsigned int square_of_sum(unsigned int number);
+unsigned int difference(unsigned int number);
```

Comment:
Given that the exercise is "difference of squares", why not name this function `difference_of_squares`?